tests/test-cases/Group9-VIC-Admin/9-2-VICAdmin-CertAuth.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation  Test 9-2 - VICAdmin CertAuth
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  certs=${true}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+Default Tags
+
+*** Test Cases ***
+Wait
+    Run  sleep 30
+
+Display HTML
+    ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}
+    Log To Console  ${output}
+    Should contain  ${output}  <title>VCH Admin</title>
+
+Get Portlayer Log
```

Comment:
you need negative testing here as well - same set of tests with:
1. no certificate provided
2. a certificate provided that isn't signed by the correct CA